diff --git a/dist/changesets-get-dependents-graph.cjs.js b/dist/changesets-get-dependents-graph.cjs.js
--- a/dist/changesets-get-dependents-graph.cjs.js
+++ b/dist/changesets-get-dependents-graph.cjs.js
@@ -35,7 +35,7 @@ const getValidRange = potentialRange => {
     return null;
   }
   try {
-    return new Range__default["default"](potentialRange);
+    return new Range__default["default"](potentialRange, {includePrerelease: true});
   } catch (_unused) {
     return null;
   }
